www/jury/config.php

```diff
@@ -84,56 +84,69 @@
     "<tr class=\"thleft\"><th>Option</th><th>Value(s)</th><th>Description</th></tr>\n" .
     "</thead>\n<tbody>\n";
 
-$extra = ' class="config_input"';
```

Comment:
This whole file can be removed now right?

Reply:
Done.